rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -499,11 +499,17 @@ private InetAddress getMyPublicIpFromRemoteService() {
         }
 
         InetAddress bindAddress = getBindAddress();
-        if (bindAddress.isAnyLocalAddress()) {
-            throw new RuntimeException("Wildcard on bind address it's not allowed as fallback for public IP " + bindAddress);
+        if (!bindAddress.isAnyLocalAddress()) {
+            return bindAddress;
         }
 
-        return bindAddress;
+        try {
+            // workaround to provide a last resource fallback to calculate node's public IP
```

Comment:
perhaps `last resort` ?

Reply:
Yep, literal translation here :), will fix it.